tt_metal/llrt/tt_cluster.cpp

```diff
@@ -973,18 +973,21 @@ std::tuple<tt_cxy_pair, tt_cxy_pair> Cluster::get_eth_tunnel_core(
 }
 
 // TODO: ALLAN Can change to write one bit
-void Cluster::set_internal_routing_info_for_ethernet_cores(bool enable_internal_routing) const {
+void Cluster::set_internal_routing_info_for_ethernet_cores(bool enable_internal_routing, const std::vector<chip_id_t> &target_mmio_devices) const {
```

Comment:
we call this function in a number of different spots for both init and close. Do we need to survey whether those callsites need update?

Is it simpler to just read back the enable bit before writing ?